Sources/TeslaSwift/Model/DriveState.swift

```diff
@@ -10,8 +10,14 @@ import Foundation
 import CoreLocation
 
 open class DriveState: Codable {
+    public enum ShiftState: String, Codable {
+        case drive = "D"
+        case parked = "P"
+        case reverse = "R"
+        case neutral = "N"
+    }
 	
-	open var shiftState: String?
+	open var shiftState: ShiftState?
```

Comment:
breaking change

Reply:
alternatively, we could keep `shiftState` and introduce `state` which is just `ShiftState`

Reply:
This is fine